libs/performance_counters/include/hpx/performance_counters/performance_counter_set.hpp

```diff
@@ -12,10 +12,10 @@
 #include <hpx/errors.hpp>
 #include <hpx/lcos/dataflow.hpp>
 #include <hpx/lcos/future.hpp>
+#include <hpx/pack_traversal/unwrap.hpp>
```

Comment:
This module now depends on pack_traversal

Reply:
This is another thing that we could check in CI (#4497).